storybook/src/docs/app-icon.docs.mdx

```diff
@@ -0,0 +1,152 @@
+import { Meta } from "@storybook/blocks";
+import { IcoFavicon, PngIconAppleTouch, PngIconsWebApp, SvgIconDefault } from "./components/AppIcons";
+
+<Meta title="Docs/Assets/Favicon & App Icons" />
+
+# Favicon & App Icons
+
+Examples of all the assets needed to display a website icon (a favicon) in the web browser or publish a web app with the proper app names and icons. This set should cover all the combinations of common operating systems (Android, iOS, macOS, Windows) and browsers (Chrome, Safari, Edge, Firefox).
+
+## Overview & examples
+
+### Default SVG Icon
+
+<SvgIconDefault />
+
+```html
+<link rel="icon" type="image/svg+xml" href="icon.svg" />
+```
+
+### Favicon
+
+<IcoFavicon />
+
+```html
+<link rel="icon" href="favicon.ico" sizes="16x16 32x32 64x64" />
```

Comment:
This other [definitive article on how to favicon in 2024](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) 😅 suggests only 32px. Elsewhere, I read that this set of 16, 32 and 64 covers almost all needs.

Reply:
So from what I understand this about Google Search results displaying the `*.ico` and that it can only handle sizes that are a multiple of 48px. Seems weird that Google can't handle more filetypes and sizes? 🤔

Should we base our decisions for the website icon on what a website (google.com) does with it?

Reply:
Well… it _is_ a popular website, no?

Is it about the `.ico` file? Does it contain the image in three sizes already? If so we can add a fourth.

Reply:
I added the 48px resolution to the ico file and in the link's sizes attributes.